teos/teosd.py

```diff
@@ -227,28 +235,28 @@ def main(config):
     command_line_conf = {}
     data_dir = DATA_DIR
 
+    opts, _ = getopt(
+        argv[1:],
+        "hd",
+        [
+            "apibind=",
+            "apiport=",
+            "rpcbind=",
+            "rpcport=",
+            "btcnetwork=",
+            "btcrpcuser=",
+            "btcrpcpassword=",
+            "btcrpcconnect=",
+            "btcrpcport=",
+            "btcfeedconnect=",
+            "btcfeedport=",
+            "datadir=",
+            "wsgi=" "daemon",
```

Comment:
missing comma

Reply:
Fixed in 54e7f24e79d8e9497fe543cfea839b6c7677453e